donation/migrations/0001_initial.py

```diff
@@ -20,6 +23,8 @@ class Migration(migrations.Migration):
                 ('quantity', models.IntegerField(default=0)),
                 ('location', models.CharField(max_length=200)),
                 ('time', models.DateTimeField(db_index=True, default=django.utils.timezone.now)),
+                ('priority', models.CharField(choices=[('HIGH', 'High'), ('MEDIUM', 'Medium'), ('LOW', 'Low')], max_length=6)),
+                ('created_by', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='donation_requests', to=settings.AUTH_USER_MODEL)),
```

Comment:
we don't actually edit the migrations, if there are any changes in the model you should run `makemigrations` and it'll create a new migration with changes